build/webpack.js

```diff
@@ -57,6 +53,15 @@ export default async function getBaseWebpackConfig (dir, {dev = false, isServer
     babel: {
       loader: 'next-babel-loader',
       options: {dev, isServer}
+    },
+    hotSelfAccept: {
+      loader: 'hot-self-accept-loader',
+      options: {
+        include: [
+          path.join(dir, 'pages')
+        ],
+        extensions: /\.(js|jsx)$/
```

Comment:
nitpick, but how about `jsx?` 🏌️‍♂️

Reply:
This is intended to not confuse people reading the code.